config/routes.rb

```diff
@@ -4,6 +4,8 @@
   devise_for :users
   root 'static_pages#home'
   get 'kontakt' => 'static_pages#kontakt'
+  get 'kontakt' => 'static_pages#new'
+  post 'kontakt' =>  'static_pages#create'
```

Comment:
static_pages#new and static_pages#create doesnt suggest it's related contact form